content/applications/websites/ecommerce/products.rst

```diff
@@ -4,266 +4,341 @@
-  your website. If you have multiple websites, you can select a different one per website;
-- :guilabel:`Out-of-Stock (Continue Selling)`: enabling it allows customers to continue placing
-  orders even when the product is **out-of-stock**.
-  Leave it  unchecked to **prevent orders**;
-- :guilabel:`Show Available Qty`: enabling it displays the available quantity left under a specified
-  threshold on the product page. The available quantity is calculated based on the 'On hand'
-  quantity minus the quantity already reserved for outgoing transfers.
-
-.. seealso::
-   :ref:`Allow only selected customers to buy <cart/prevent-sale>`
+In the :guilabel:`Inventory Defaults` sub-section, select your eCommerce selling strategy:
 
-Selling as kit
---------------
-
-If you are selling non-prepackaged kits (i.e., the kits are made of individual products), we
-recommend you read the related documentation to keep track of your stock.
+- :guilabel:`Out-of-Stock`: enable :guilabel:`Continue Selling` to allow customers to continue
+  placing orders even when the product is **out-of-stock**. Leave the box unchecked to
+  **prevent orders**;
+- :guilabel:`Show Available Qty`: displays the available quantity left under a specified threshold
+  on the product page. The available quantity is calculated based on the 'On hand' quantity minus
+  the quantity already reserved for outgoing transfers.
 
-.. seealso::
-   :doc:`../../inventory_and_mrp/manufacturing/advanced_configuration/kit_shipping`
+.. _products/product-comparison:
 
 Product comparison
 ==================
 
-You can enable a **product comparison tool** for your eCommerce by going to
-:menuselection:`Website --> Configuration --> Settings --> Shop - Products`, and ticking
-:guilabel:`Product Comparison Tool`. This tool allows to save products' **specifications** and
-compare them against each other on a single page.
-
-On the product page, scroll down to the :guilabel:`Specifications` section and click
-:guilabel:`Compare`. Repeat the same process for all products you wish to compare. Then, click the
-:guilabel:`Compare` button of the pop-up window at the bottom of the page to reach the comparison
-summary.
+Your visitors can compare products based on their attributes when the :guilabel:`Product Comparison
+Tool` is enabled. You can activate it by going to :menuselection:`Website --> Configuration -->
+Settings --> Shop - Products` section.
 
 .. note::
    The :guilabel:`Product Comparison Tool` can only be used if :doc:`attributes <products/variants>`
-   are set on the **product's template**.
+   are set on the :ref:`product form <products/product-form>`.
+
+On your main shop page, click the :icon:`fa-exchange` (:guilabel:`Compare`) option on the products
+you want to compare. Then, click :icon:`fa-exchange` :guilabel:`Compare` in the pop-up window at the
+bottom of the page to reach the comparison summary.
```

Comment:
Should we also add that you can click Compare on a product page (in a tip for example)? not sure, what do you think?

Reply:
added in a note with the link to the Customization section